src/App.js

```diff
@@ -1,109 +1,120 @@
-import React from "react";
+import React, { Fragment } from "react";
 import ReactDom from "react-dom/client";
-import logo from "./Img/logo.jpg"
-import userIcon from "./Img/userIcon.jpg"
-
-//Start of Left Column
-//Nesting Elements in React
-const nestingEleInReactHeading = React.createElement("div",{id:"title"},[
-    React.createElement("h1",{},"Namaste React BootCamp"),
-    React.createElement("h2",{},"from zero to hero"),
-    React.createElement("h3",{},"in 3 months")]);
-
-const nestingEleInReactRoot = ReactDom.createRoot(document.getElementById("rootReact"));
-
-nestingEleInReactRoot.render(nestingEleInReactHeading);
-
-//Same in JSX
-const nestingEleInJSXHeading = (<div>
-    <h1>Namaste React BootCamp JSX</h1>
-    <h2>from zero to hero JSX</h2>
-    <h3>in 3 months JSX</h3>
-</div>);
-
-const nestingEleInJSXRoot = ReactDom.createRoot(document.getElementById("rootJSX"));
-
-nestingEleInJSXRoot.render(nestingEleInJSXHeading);
-
-//Passing attributes
-const passingAttrHeading = (<div>
-    <h1 id="passAttrId">Passing Attribute ID</h1>
-    <h2 className="passAttr-Class">Passing Attributes Class</h2>
-    <h2 tabIndex="1">Passing Attributes Tab Index</h2>
-</div>);
-
-const passingAttrRoot = ReactDom.createRoot(document.getElementById("rootPassAttr"));
-
-passingAttrRoot.render(passingAttrHeading);
-
-//Composition of components
-
-//TitleComponent is called as functional component    
-const TitleComponent = () =>
+import CardComponent from "./CardComponent.js";
+import sowmyaImg from "./Img/Sowmya.png"
+
+const title = "Strom Troopers";
+
+const data=[
```

Comment:
you can make a separate file for db.

Reply:
moving into separate file is in next assignment